js/src/index.ts

```diff
@@ -3,9 +3,8 @@ import { ImageViewerApp } from "@aics/web-3d-viewer";
 import React from "react";
 import ReactDOM from "react-dom";
 
-import "antd/dist/antd.css";
-// TODO: this starts as .less and needs to be converted to .css in tsc step
-import "./style.css";
+// Imported after tsc transpile - links out of lib, back into src
+import "../src/style.less";
```

Comment:
TBH I think this looks a bit clunky - the path should be one that makes sense within this file. 
My preference would be to have a build step that simply copies the .less file into the `lib/` dir.  For example, reinstate the `build:css` step but just have it copy the file over.
And in the comment here, I'd say that we are using webpack's less-loader so we don't need to transpile this until webpack runs.

Reply:
Yeah I wasn't particularly satisfied with this solution either. Does it make sense to move `style.less` into the `styles` directory and copy that directory over into `lib/`? Otherwise we create the same problem one level down, where imports in `style.less` have to reach back out of `lib/` and into `src/`. Unfortunately no solution will be 100% nice without all the building happening in a single step, which I couldn't quite get working.

Reply:
if the import is `import "./style.less"` then it will work in the context of reading this source code, and also in teh context of running webpack on the contents of the lib/ directory.  But that then requires that something copies the file into lib/ (aka package.json scripts).
I just feel that it's cleaner if it says `import "./style.less"` .  If you want to put it in a `styles` subdirectory that would be totally fine too.